pipelines/toast_satellite_sim.py

```diff
@@ -243,6 +296,32 @@ def main():
                         help='Solar system speed galactic longitude[degrees]',
                         type=float, default=263.99)
 
+    # Conviqt beam convolution
+
+    parser.add_argument( "--run_conviqt", required=False, default=False,
+        action="store_true", help="Convolve map with full beam alms" )
+    parser.add_argument( "--conviqt_beamfile_alm", required=False, default="beam_DETECTOR_alm.fits",
+        help="Full beam alms filename, they should be defined in the same reference frame"
+        "both for A and B detectors, i.e. no rotation, not even for the polarization angle is"
+        "performed by TOAST."
```

Comment:
@keskitalo about the reference frame, I would start now supporting just 1. which is currently used by Karl for PICO that defines all the beams in the same reference frame, see https://github.com/zonca/pico-simulations/issues/17